.travis/run_python_tests.sh

```diff
@@ -11,7 +11,8 @@ cd "${TRAVIS_BUILD_DIR}/exonum-java-binding"
 
 # Generate protobuf files needed for plugins
 source ./tests_profile
```

Comment:
RUSTFLAGS=none , otherwise it is likely to fail with no Rust

Reply:
:thinking: I don't think it'll help actually, we're always overwrite RUSTFLAGS

Reply:
Ah, actually it works, because our tests_profile does not fail on errors